perf/benchmark/runner/runner.py

```diff
@@ -281,24 +290,62 @@ def kubectl_cp(from_file, to_file, container):
     run_command_sync(cmd)
 
 
+def run_nighthawk(pod, remote_cmd, labels):
+    # Use a local docker instance of Nighthawk to control nighthawk_service running in the pod
+    # and run transforms on the output we get.
+    docker_cmd = "docker run --network=host {docker_image} {remote_cmd}".format(
+        docker_image=NIGHTHAWK_DOCKER_IMAGE, remote_cmd=remote_cmd)
+    print(docker_cmd, flush=True)
+    process = subprocess.Popen(shlex.split(docker_cmd), stdout=subprocess.PIPE)
+    (output, err) = process.communicate()
+    exit_code = process.wait()
+
+    if exit_code == 0:
+        with tempfile.NamedTemporaryFile(dir='/tmp', delete=True) as tmpfile:
+            dest = tmpfile.name
+            with open("%s.json" % dest, 'wb') as f:
+                f.write(output)
+            print("Dumped Nighthawk's json to {dest}".format(dest=dest))
+
+            # Send human readable output to the command line.
+            os.system(
+                "cat {dest}.json | docker run -i {docker_image} nighthawk_output_transform --output-format human".format(docker_image=NIGHTHAWK_DOCKER_IMAGE, dest=dest))
```

Comment:
Could you use `--rm` flags for `docker run`?

Reply:
Done: e9ed651b4056258a543c8da4476a34ab0c72edc0